doc/code/qml_compiler.rst

```diff
@@ -0,0 +1,8 @@
+qml.compiler
+===============
+
+.. currentmodule:: pennylane.compiler
+
+.. automodapi:: pennylane.compiler
+    :no-heading:
+    :include-all-objects:
```

Comment:
I thought this is part of the style lol

Reply:
Unfortunately we have two styles :(

- For some module docstrings we have more curated function summary tables (`autosummary`)
- For others, we don't care as much and leave it to be automatic. (`automodapi`)

We don't want to use both at the same time though, as it will cause functions to be duplicated in the docs.

Reply:
Thanks for the info!